src/test/java/streams/part2/exercise/Exercise1.java

```diff
@@ -0,0 +1,164 @@
+package streams.part2.exercise;
+
+import lambda.data.Employee;
+import lambda.data.JobHistoryEntry;
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.assertEquals;
+
+@SuppressWarnings({"ConstantConditions", "unused"})
+public class Exercise1 {
+
+    @Test
+    public void calcTotalYearsSpentInEpam() {
+        List<Employee> employees = getEmployees();
+
+        // TODO реализация
+        Long hours = employees.stream().
+                map(Employee::getJobHistory).
+                flatMap(Collection::stream).
+                filter(jobHistoryEntry -> jobHistoryEntry.getEmployer().equals("EPAM")).
+                mapToLong(JobHistoryEntry::getDuration).
+                sum();
```

Comment:
As is customary in Java: chain calls dots are placed at the beginning of a new line.

Reply:
fixed a4bdca1e71010142d19c2c7129804a9196327657